application/backend/app/alembic/versions/da385d690aae_schema.py

```diff
@@ -120,7 +125,8 @@ def upgrade() -> None:
         sa.ForeignKeyConstraint(["project_id"], ["projects.id"], ondelete="CASCADE"),
         sa.ForeignKeyConstraint(["source_id"], ["sources.id"], ondelete="SET NULL"),
         sa.PrimaryKeyConstraint("id"),
-        sa.Index("idx_dataset_items_project_created_at", "project_id", "created_at"),
+        sa.Index("idx_dataset_items_user_reviewed", "project_id", "user_reviewed"),
+        sa.Index("idx_dataset_items_source", "source_id"),
```

Comment:
We don't have plans yet to filter dataset items by source id, but it could be a nice future addition. I suggest to remove this index for now.